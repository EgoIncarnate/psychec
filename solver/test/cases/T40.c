// struct names created by the solver follow the convencion Tx, where x
// is an integer (although the exact ordering is undefined)

void f() {
    a.x = 6.6;
    b->y = 1.0;
    c->z = "foo";
    c->w = 10;
    d->next = d;
    d->val = 9.0;
    return;
}
