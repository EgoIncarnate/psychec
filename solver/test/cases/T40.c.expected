struct __T0__ {double x;} a ;
struct __T1__ {double y;}* b ;
struct __T2__ {char* z;int w;}* c ;
struct __T3__ {double val;struct __T3__ * next;}* d ;
